package.json

```diff
@@ -64,18 +64,22 @@
     "babel-loader": "^8.0.4",
     "file-loader": "^6.0.0",
     "json-loader": "^0.5.7",
+    "leaflet": "^1.7.1",
```

Comment:
The link tag is needed  for the https://github.com/comunica/jQuery-Widget.js/pull/86#discussion_r659560794 
I have removed the script tag as it picks javascript included in the node_modules

Reply:
Are you sure the link's CSS file is not available within the `leaflet` dependency? Perhaps it can just be reused from there? So that the link tag refers to something within our own package?

Reply:
Oh yes , did find it in the leaflet dependency. Thanks